lib/physiqual/data_services/summarized_data_service.rb

```diff
@@ -112,6 +110,10 @@ def cluster_in_buckets(data, from, to)
           # TODO: Just flatten whatever you add before adding it and check that your concat works correctly.
           buckets[current_bucket][values_field] = buckets[current_bucket][values_field].flatten
```

Comment:
ik weet dat ik dit heb geschreven, maar is `buckets[current_bucket][values_field].flatten!` niet hetzelfde?:)